onnx_tf/handlers/backend/conv_mixin.py

```diff
@@ -97,8 +95,8 @@ def conv(cls, node, input_dict, transpose=False):
     # Is this convolution depthwise we can support?
     depthwise = (x_rank == 4 and len(weight_shape) == 4 and group != 1 and
                  not transpose and not (None in weight_shape))
-    if depthwise and isinstance(x_shape, np.ndarray):
-      depthwise = bool(group == x_shape[1])
+    if depthwise and (x_rank > 1 and x.get_shape().as_list()[1] != None):
```

Comment:
It might not be necessary to check x_rank > 1 since depthwise is true, meaning x_rank == 4

Reply:
Yes. Thanks. Remove x_rank > 1.